appbars/src/main/java/com/decathlon/vitamin/compose/appbars/bottomnavigations/VitaminBottomNavigations.kt

```diff
@@ -26,8 +27,12 @@ object VitaminBottomNavigations {
      * @param modifier The [Modifier] to be applied to this BottomNavigation
      * @param colors The colors of the background and the content elements in selected and unselected mode
      */
+    @Deprecated(
```

Comment:
When deprecating a method, you can safely remove the KDoc from that method, it puches the developer to use the new method since the old one is not documented anymore.

PS: It's an opinion, I don't know any official rule about it

Reply:
Don't have any opinion about this. I just checked quickly in Compose and they keep kdoc but it is probably not very important to keep or not.

Reply:
You're right, we do need to discuss and write our depreciation strategy that we will follow on the next major version of Vitamin. 
For this current version, I would keep the documentation even on deprecated methods.